ibm/service/kms/data_source_ibm_kms_keys.go

```diff
@@ -199,6 +203,7 @@ func dataSourceIBMKMSKeysRead(d *schema.ResourceData, meta interface{}) error {
 		keyInstance["name"] = key.Name
 		keyInstance["crn"] = key.CRN
 		keyInstance["standard_key"] = key.Extractable
+		keyInstance["description"] = key.Description
 		keyInstance["aliases"] = key.Aliases
 		keyInstance["key_ring_id"] = key.KeyRingID
 		keyMap = append(keyMap, keyInstance)
```

Comment:
Noticed a gap here where we don't set key policy information when `alias` is given for this block (compared with when `key_id` is given for the next block.